src/main.rs

```diff
@@ -1,20 +1,49 @@
+use rand::seq::SliceRandom;
+use rand::thread_rng;
+use std::time::{SystemTime, UNIX_EPOCH};
+
 fn main() {
-    println!("Hello, world!");
-    println!("Hello, world!");
+    let now = SystemTime::now();
+    match now.duration_since(UNIX_EPOCH) {
+        Ok(duration) => {
+            let seconds_since = duration.as_secs();
+            if seconds_since % 5 == 0 {
+                println!("UWUUUUUUUUUUUUUUUUUUUUU");
+            } else {
+                println!("YOU HAVE COMMITTED HERESY!");
+            }
+        }
+        Err(_) => {
+            println!("Error occurred while calculating duration.");
+        }
+    }
+    println!("Answering incorrectly results in DEATH!");
+    println!("~uWu~");
 
-    let question = "What is the answer to life, the universe, and everything?".to_string();
+    let question = "What is the answer to life, the universe, and everything in it?".to_string();
 
-    if the_answer(question) == 42 {
-        println!("Correct!");
-    } else {
-        println!("Incorrect!");
-    }
+    let outcome = match the_answer(question) {
+        42 => "Correct!",
+        69 => "Yes!",
+        9001 => "Such a high power level is obviously correct!",
+        _ => "Incorrect!",
+    };
+    println!("Outcome: {}", outcome);
 }
 
 fn the_answer(question: String) -> i32 {
-    let base = "42".to_string();
-    if question == "What is the answer to life, the universe, and everything?" {
-        return base.parse().unwrap();
+    if question == "What is the answer to life, the universe, and everything in it?" {
+        return draw_answer_from_hat();
     }
     0
 }
+
+fn draw_answer_from_hat() -> i32 {
+    let numbers = [1, 2, 3, 42, 69, 9001];
+    let mut rng = thread_rng();
```

Comment:
Please use xoshiro rng generators for this project

Reply:
That sounds like an absolute, only a Sith deals in absolutes.